doc/examples/qti_simulation_tutorial.py

```diff
@@ -0,0 +1,137 @@
+"""
+============================
+Q-SpaceTrajectory Simulation
+============================
+
+The most basic assumption of qti states that tissues can be denoted by multiple gaussian component.
+QTI can only handle a single gtab at a time, so in this tutorial we'll start by taking 2 gtabs then combine them into a single btensor. 
+"""
+
```

Comment:
Before this lines of comments. You should say something like: "Let's first import all relevant code for this simulations"